.changeset/funny-pianos-grow.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-catalog-react': patch
+---
+
+Fixed bug in `EntityTagPicker` that filtered on unavailable tags for the selected kind.
```

Comment:
From my understanding we only update `EntityTagPicker` or did I miss something? I think we should remove the other lines below if that's the case :)

Reply:
I updated the other pickers in a different PR that was merged before I could add a changeset. I can totally remove these lines I was just hoping we could add them in since they were not added before. This is the other pr https://github.com/backstage/backstage/pull/15058

Reply:
Oh it looks like that indeed snuck in without a changeset, thanks for pointing it out @sarahpmedeiros! 

It has however already been released, in that case could you instead update the 1.2.2 section in https://github.com/backstage/backstage/blob/master/plugins/catalog-react/CHANGELOG.md#122 and add a note about the previous fixes in the list of patch changes.

And then just have this changeset only mention `EntityTagPicker`

Reply:
will do, updated this changeset thank you!

Reply:
https://github.com/backstage/backstage/pull/15377 also created the other pr